src/features/content-answers/__tests__/ContentAnswersOpenButton.test.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+
+import ContentAnswersOpenButton from '../ContentAnswersOpenButton';
+
+import messages from '../messages';
+
+describe('features/content-answers/ContentAnswersOpenButton', () => {
+    const renderComponent = (props: {}) => render(<ContentAnswersOpenButton onClick={jest.fn()} {...props} />);
+
+    test('should call the onClick callback', () => {
+        const onClick = jest.fn();
+        const wrapper = renderComponent({ onClick });
+        expect(onClick).toBeCalledTimes(0);
+
+        fireEvent.click(wrapper.getByTestId('bdl-ContentAnswersOpenButton'));
+        expect(onClick).toBeCalledTimes(1);
+    });
+
+    test('should display correct tooltip', () => {
+        const wrapper = renderComponent({});
+        fireEvent.mouseOver(screen.getByTestId('bdl-ContentAnswersOpenButton'));
+        expect(wrapper.getByText(messages.defaultTooltip.defaultMessage)).toBeInTheDocument();
```

Comment:
sorry, I miscommunicated this in the previous comment. you can keep the `content-answers-open-button` id if you'd like

what I meant was the the `wrapper` part is not needed if you're using `screen`
```suggestion
        renderComponent();
        fireEvent.mouseOver(screen.getByTestId('bdl-ContentAnswersOpenButton'));
        expect(screen.getByText(messages.defaultTooltip.defaultMessage)).toBeInTheDocument();
```
and same for the test above where you no longer need `wrapper`